injective/assetlist.json

```diff
@@ -1334,6 +1334,80 @@
         "twitter": "https://x.com/neptune_finance"
       },
       "type_asset": "sdk.coin"
+    },
+    {
+      "description": "Factory wrapped variant of the CW20 nUSDT, the receipt token for lent USDT on Neptune Finance",
+      "denom_units": [
+        {
+          "denom": "factory/inj14ejqjyq8um4p3xfqj74yld5waqljf88f9eneuk/inj1cy9hes20vww2yr6crvs75gxy5hpycya2hmjg9s",
+          "exponent": 0
+        },
+        {
+          "denom": "nUSDT",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/inj14ejqjyq8um4p3xfqj74yld5waqljf88f9eneuk/inj1cy9hes20vww2yr6crvs75gxy5hpycya2hmjg9s",
+      "name": "Factory Wrapped Neptune Receipt nUSDT",
+      "display": "nUSDT",
+      "symbol": "nUSDT",
+        "traces": [
+        {
+          "type": "Factory-Wrap",
+          "counterparty": {
+            "chain_name": "injective",
+            "base_denom": "inj1cy9hes20vww2yr6crvs75gxy5hpycya2hmjg9s"
+          },
+          "provider": "CW20-Wrapper"
+        }
+      ],
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/nusdt.png",
+          "theme": {
+            "primary_color_hex": "#54ac94"
+          }
+        }
+      ],
+      },
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "Factory wrapped variant of the CW20 nUSDC, the receipt token for lent USDC on Neptune Finance",
+      "denom_units": [
+        {
+          "denom": "factory/inj14ejqjyq8um4p3xfqj74yld5waqljf88f9eneuk/inj1dafy7fv7qczzatd98dv8hekx6ssckrflswpjaz",
+          "exponent": 0
+        },
+        {
+          "denom": "nUSDC",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/inj14ejqjyq8um4p3xfqj74yld5waqljf88f9eneuk/inj1dafy7fv7qczzatd98dv8hekx6ssckrflswpjaz",
+      "name": "Factory Wrapped Neptune Receipt nUSDC",
+      "display": "nUSDC",
+      "symbol": "nUSDC",
+        "traces": [
+        {
+          "type": "Factory-Wrap",
```

Comment:
Factory-Wrap is not a defined type. Please use one of the defined types, such as "wrapped"